MonoGame.Framework/SDL/SDLGamePlatform.cs

```diff
@@ -0,0 +1,212 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework.Input;
+
+namespace Microsoft.Xna.Framework
+{
+    internal class SdlGamePlatform : GamePlatform
+    {
+        public override GameRunBehavior DefaultRunBehavior
+        {
+            get { return GameRunBehavior.Synchronous; }
+        }
+
+        private readonly Game _game;
+        private readonly OpenALSoundController _soundControllerInstance;
+        private readonly List<Keys> _keys;
+
+        private int _isExiting;
+        private SdlGameWindow _view;
+
+        public SdlGamePlatform(Game game)
+            : base(game)
+        {
+            _game = game;
+            _keys = new List<Keys>();
+            Keyboard.SetKeys(_keys);
+
+            Sdl.Version sversion;
+            Sdl.GetVersion(out sversion);
+
+            Sdl.Major = sversion.Major;
+            Sdl.Minor = sversion.Minor;
+            Sdl.Patch = sversion.Patch;
+
+            var version = 100 * Sdl.Major + 10 * Sdl.Minor + Sdl.Patch;
+
+            if (version <= 204)
+                Debug.WriteLine ("Please use SDL 2.0.5 or higher.");
```

Comment:
Is 2.0.5 a typo or does it mean that 2.0.5 (unstable) is the target? (because ```version <= 204``` allows 2.0.4).